src/js/game/LevelMVC/LevelModel.js

```diff
@@ -1116,6 +1196,106 @@ module.exports = class LevelModel {
     }
   }
 
+  determineRedstoneSprite(x, y) {
+        let foundAbove = false;
+        let foundBelow = false;
+        let foundRight = false;
+        let foundLeft = false;
+        let myIndex = (this.yToIndex(y) + x);
+        let belowIndex = (this.yToIndex(y + 1)) + x;
+        let aboveIndex = (this.yToIndex(y - 1)) + x;
+        let leftIndex = (this.yToIndex(y)) + x - 1;
+        let rightIndex = (this.yToIndex(y)) + x + 1;
+
+        let borderCount = 0;
+
+        if (!this.inBounds(x, y)) {
+            // If we're looking out of bounds, just leave, you goofball.
+            return;
+        }
+        // If in bounds, we want to see if any redstone is around the index in question
```

Comment:
Can we re-use `getOrthogonalBlocks` here?  Once you merge latest `master` into this branch you should have access to `LevelPlane.prototype.getOrthogonalBlocks`.